webapp/src/components/Notification/Notification.js

```diff
@@ -15,6 +15,17 @@ import useMediaQuery from '@material-ui/core/useMediaQuery'
 import { useTranslation } from 'react-i18next'
 import { useLocation } from 'react-router-dom'
 
+
```

Comment:
Remember, you just need one line to separate import groups

Reply:
Disculpa @tetogomez , aún no era la versión final, es que Leister me lo había solicitado para utilizar una parte del código, por lo implemente la etiquete WIP, gracias por las observaciones!!